Spec/Test Utilities/TestUtilities.swift

```diff
@@ -360,6 +360,17 @@ class AblyTests {
         let items = jsonItems.map{ $0 }.map(CryptoTestItem.init)
         return (keyData, ivData, items)
     }
+    
+    class func backoffWithJitterDelaysForTimeout(_ timeout: TimeInterval) -> [TimeInterval] {
+        // This is calculated for seeding number = 5 (randomFraction gives [ 0.75, 0.65, 0.1, 0.72, 0.76 ])
```

Comment:
From [reading the documentation for `random`](https://pubs.opengroup.org/onlinepubs/007908799/xsh/initstate.html) it's not very clear to me whether `random` is guaranteed to always return the same numbers for all implementations (i.e. whether the algorithm could change in an OS update). But I don't think it's a huge problem, we can cross that bridge if we need to later.